### PR TITLE
challenge solved to chapter 7

### PR DESCRIPTION
**Pull request by dipushrestha**
practice on java constructors, composition, encapsulation and polymorphism

### CODE REVIEW

File: dipendra/composition/Ceiling.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
+package composition;
+
+import java.awt.*;
```
Comment (shradayshakya) [inline dipendra/composition/Ceiling.java]: Do we need to add the entire package?

Comment (silwalanish) [inline dipendra/composition/Ceiling.java]: Unused import

File: dipendra/composition/Door.java [hunk 2]
```diff
@@ -0,0 +1,29 @@
+package composition;
+
+import java.awt.*;
```
Comment (silwalanish) [inline dipendra/composition/Door.java]: Unused import

File: dipendra/composition/Floor.java [hunk 3]
```diff
@@ -0,0 +1,29 @@
+package composition;
+
+import java.awt.*;
```
Comment (silwalanish) [inline dipendra/composition/Floor.java]: unused import

File: dipendra/composition/Furniture.java [hunk 4]
```diff
@@ -0,0 +1,39 @@
+package composition;
+
+import java.awt.*;
```
Comment (silwalanish) [inline dipendra/composition/Furniture.java]: .

File: dipendra/composition/Main.java [hunk 5]
```diff
@@ -0,0 +1,29 @@
+package composition;
+
+import java.awt.*;
```
Comment (silwalanish) [inline dipendra/composition/Main.java:3]: .

File: dipendra/composition/Wall.java [hunk 6]
```diff
@@ -0,0 +1,39 @@
+package composition;
+
+import java.awt.*;
```
Comment (silwalanish) [inline dipendra/composition/Wall.java]: .

File: dipendra/composition/Window.java [hunk 7]
```diff
@@ -0,0 +1,39 @@
+package composition;
+
+import java.awt.*;
```
Comment (rajshreeee) [inline dipendra/composition/Window.java]: has this package been used anywhere?

Comment (silwalanish) [inline dipendra/composition/Window.java]: .

File: dipendra/polymorphism/Main.java [hunk 8]
```diff
@@ -0,0 +1,127 @@
...
+    public void accelerate() {
+        System.out.println("Accelerated");
+    }
+
+    public void brake() {
+        System.out.println("Braked");
+    }
+}
+
+class Toyota extends Car {
```
Comment (silwalanish) [inline dipendra/polymorphism/Main.java:37]: Suggestion: consider making different files for each class.

**Review by silwalanish — COMMENTED** [review]
Remove unnecessary imports

File: dipendra/composition/Ceiling.java (hunk 1 above)
Comment (dipushrestha) [reply dipendra/composition/Ceiling.java]: issue resolved

File: dipendra/composition/Window.java (hunk 7 above)
Comment (dipushrestha) [reply dipendra/composition/Window.java]: yes awt includes Dimension class

File: dipendra/composition/Ceiling.java (hunk 1 above)
Comment (dipushrestha) [reply dipendra/composition/Ceiling.java]: used for Dimension class
clearified with -> import java.awt.Dimension

File: dipendra/composition/Door.java (hunk 2 above)
Comment (dipushrestha) [reply dipendra/composition/Door.java]: resolved

File: dipendra/polymorphism/Main.java (hunk 8 above)
Comment (dipushrestha) [reply dipendra/polymorphism/Main.java:37]: excercise rule mentioned to make all class in one file

**Pull request merged**